### PR TITLE
LRQA-77964 | master

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/LRQA-77964
Submit for master only.

cc/ @JoyceWang08211, @gaborlovasdotcom

### CODE REVIEW

**Comment by jaydawu** [comment]
Hi @JoyceWang08211 and @gaborlovasdotcom,

The reason why I used Navigation to URL [adding the widget template page manually](https://github.com/liferay-staging/liferay-portal/pull/160/files#diff-135007ffd8bf05730d1eacbaa527fbf2a84f852e021d568e05b8a030963240e6R73-R79) is that it cannot be added via JSON under the new user. The JSON add using the wrong default "Test" user(like the below pic), I tried to edit, but it didn't work. If you guys have a better way to add it, please tell me. Thanks!

Error

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/widgetpagetemplate/WidgetPageTemplatesWithStaging.testcase [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Site Name");
+		}
+	}
+
+	@description = "This is a test for LPS-146888. Assert that can publish a widget page template created by another deleted admin user on local staging"
+	@priority = "3"
+	test CanPublishAWidgetPageTemplateCreatedByAnotherDeletedUserOnLocalStaging {
+		task ("Given: Enable local staging with a new blank site") {
+			JSONStaging.enableLocalStaging(groupName = "Site Name");
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/widgetpagetemplate/WidgetPageTemplatesWithStaging.testcase:39]: Please move this step into the set-up part.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/widgetpagetemplate/WidgetPageTemplatesWithStaging.testcase [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "userea@liferay.com");
+
+			JSONGroup.assignUserToGroup(
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/widgetpagetemplate/WidgetPageTemplatesWithStaging.testcase:54]: Useless step. No need to assign admin user to site.

**Review by JoyceWang08211 — CHANGES_REQUESTED** [review]

**Comment by JoyceWang08211** [comment]
Just tiny changes are needed. Well done @jaydawu

**Comment by JoyceWang08211** [comment]
ci:test:staging:nocompile

**Comment by jaydawu** [comment]
Already updated! Please review, thanks :)

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by JoyceWang08211** [comment]
ci:forward

**Pull request closed without merging**